pkg/util/signal/signal_posix.go

```diff
@@ -32,7 +32,7 @@ func SetupSignalHandler(shutdownFunc func()) {
 
 	signal.Notify(usrDefSignalChan, syscall.SIGUSR1)
 	go func() {
-		buf := make([]byte, 1<<16)
+		buf := make([]byte, 1<<17)
```

Comment:
```
kill -USR1 {tidb-pid}
```

This will print the goroutine stacks.
When tidb exit, and the 10080 port is closed, it's difficult to check what tidb is executing and where it hang.
So this command is useful.

The problem is that 64k is not enough for dumping the whole stacks sometimes.
So some goroutine stacks are losing ...

Increase the buffer to 128K here.